server/cmd/mmctl/commands/export.go

```diff
@@ -154,6 +155,11 @@ func exportCreateCmdF(c client.Client, command *cobra.Command, args []string) er
 		data["include_profile_pictures"] = "true"
 	}
 
+	teamId, err := command.Flags().GetString("team-id")
```

Comment:
Could this be the team name? I think it's better for being human readability, `Name` field is also a unique identifier.

Reply:
Good point, I have changed the CLI to use team names, but kept internal methods to use ids instead.